src/components/DateRangePicker.jsx

```diff
@@ -51,6 +51,7 @@ const defaultProps = {
   screenReaderInputMessage: '',
   showClearDates: false,
   showDefaultInputIcon: false,
+  showInputIconRight: false,
```

Comment:
Let's avoid using terms like "Right" - in an RTL world, "after" is more appropriate.